packages/@react-spectrum/list/stories/ListView.stories.tsx

```diff
@@ -160,70 +182,64 @@ storiesOf('ListView', module)
     </ListView>
     )
   )
-  .add('falsy ids as keys', () => (
-    <FalsyIds />
+  .add('falsy ids as keys', args => (
+    <FalsyIds {...args} />
   ))
-  .add('empty list', () => (
-    <ListView width="300px" height="300px" renderEmptyState={renderEmptyState}>
+  .add('empty list', args => (
+    <ListView width="300px" height="300px" renderEmptyState={renderEmptyState} {...args}>
       {[]}
     </ListView>
   ))
-  .add('loading', () => (
-    <ListView width="300px" height="300px" loadingState="loading">
+  .add('loading', args => (
+    <ListView width="300px" height="300px" loadingState="loading" {...args}>
       {[]}
     </ListView>
   ))
-  .add('loadingMore', () => (
-    <ListView width="300px" height="300px" loadingState="loadingMore">
+  .add('loadingMore', args => (
+    <ListView width="300px" height="300px" loadingState="loadingMore" {...args}>
       <Item textValue="Adobe Photoshop">Adobe Photoshop</Item>
       <Item textValue="Adobe Illustrator">Adobe Illustrator</Item>
       <Item textValue="Adobe XD">Adobe XD</Item>
     </ListView>
   ))
-  .add('async listview loading', () => (
-    <AsyncList />
+  .add('async listview loading', args => (
+    <AsyncList {...args} />
   ))
-  .add('async listview loading with actions', () => (
-    <AsyncList withActions />
+  .add('async listview loading with actions', args => (
+    <AsyncList withActions {...args} />
   ))
-  .add('density: compact', () => (
-    <ListView width="250px" density="compact">
-      <Item textValue="Adobe Photoshop">Adobe Photoshop</Item>
-      <Item textValue="Adobe Illustrator">Adobe Illustrator</Item>
-      <Item textValue="Adobe XD">Adobe XD</Item>
+  .add('dynamic items + renderEmptyState', args => (<EmptyTest {...args} />))
+  .add('with ActionBar', args => <ActionBarExample {...args} />)
+  .add('with emphasized ActionBar', args => <ActionBarExample isEmphasized {...args} />)
+  .add('thumbnails', args => (
+    <ListView width="250px" items={itemsWithThumbs} {...args}>
+      {(item: any) => <Item textValue={item.title}><Image src={item.url} /><Text>{item.title}</Text><Text slot="description">JPG</Text></Item>}
     </ListView>
   ))
-  .add('density: spacious', () => (
-    <ListView width="250px" density="spacious">
-      <Item textValue="Adobe Photoshop">Adobe Photoshop</Item>
-      <Item textValue="Adobe Illustrator">Adobe Illustrator</Item>
-      <Item textValue="Adobe XD">Adobe XD</Item>
+  .add('overflowMode="truncate" (default)', args => (
```

Comment:
Shouldn't this be a control and not its own story?

Reply:
Updated. Actually this uncovered a bug that the row height doesn't update when switching between overflow modes. Maybe we can address that separately though.